config/realtek_bee/bee_arm.gni

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2020 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+arm_arch = "armv8.1-m.main+dsp+mve+fp"
+
+#arm_abi = "aapcs"
```

Comment:
same here and throughout: no commented out code please. If you rely on some of these to be defaults, state so in comments and organize it as such. The code currently looks like a copy & paster with commented out parts.